module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php

```diff
@@ -279,27 +279,54 @@ public function testBulkDeleteFromList(): void
         $this->unfindCss($page, 'button[name="delete"]');
     }
 
+    /**
+     * Data provider to allow testing of top or bottom controls.
+     *
+     * @return array[]
+     */
+    public static function topOrBottomProvider(): array
+    {
+        return [
+            'top button' => [''],
+            'bottom button' => ['bottom_'],
+        ];
+    }
+
     /**
      * Test that the export control works.
      *
+     * @param string $idPrefix Prefix for bulk control IDs.
+     *
      * @return void
+     *
+     * @dataProvider topOrBottomProvider
      */
-    public function testBulkExport(): void
+    public function testBulkExport(string $idPrefix): void
     {
+        $session = $this->getMinkSession();
         $page = $this->setUpGenericBulkTest();
-        $button = $this->findCss($page, '#ribbon-export');
+        $buttonSelector = '#' . $idPrefix . 'ribbon-export';
 
         // First try clicking without selecting anything:
-        $button->click();
+        $this->clickCss($page, $buttonSelector);
         $this->checkForNonSelectedMessage($page);
         $this->closeLightbox($page, true);
 
         // Now do it for real -- we should get a lightbox prompt.
-        $page->find('css', '#addFormCheckboxSelectAll')->check();
-        $button->click();
+        $page->find('css', '#' . $idPrefix . 'addFormCheckboxSelectAll')->check();
+        $this->waitStatement('$("input.checkbox-select-item:checked").length === 2');
```

Comment:
I added this waitStatement to both of the tests so that they match other earlier tests with the same behavior. It was an early attempt to fix the click-non-registration problem. It didn't help. I'm not sure if we should leave this in or take it out.